tests/unittests/std_interoperability.cpp

```diff
@@ -1313,7 +1313,7 @@ TEST_CASE("Collection and std iterator adaptors", "[collection][container][adapt
   }
 }
 
-TEST_CASE("Collection as range", "[collection][ranges][std]") {
+TEST_CASE("Collection as range", "[collection][ranges][std][GCC11-FAIL]") {
```

Comment:
```suggestion
TEST_CASE("Collection as range", "[collection][ranges][std]") {
```
This isn't needed anymore, right? At least it isn't present in the other affected test

Reply:
Missed to remove it. Thanks a lot :)